requirements.txt

```diff
@@ -1,14 +1,34 @@
-## How to set up a Python runtime environment for Vivarium.
-#
-## Create the "vivarium" python virtual environment and select it for your project
-## directory:
-#   cd ~/code/vivarium-core
-#   pyenv local 3.8.3
-#   pyenv virtualenv vivarium
-#   pyenv local vivarium
-#
-## Install requirements
-#   pip install -r requirements.txt
-#
-
--e .
\ No newline at end of file
+astroid==2.4.2
+attrs==20.3.0
+certifi==2020.6.20
+confluent-kafka==1.5.0
```

Comment:
I think `confluent-kafka` is no longer needed.  Did you get these requirements from your environment?

Reply:
Done in ebe99a4